src/socket.ts

```
@@ -181,19 +208,31 @@ export class SocketUdp extends Socket {
+            }
+            if (err) {
+                try {
+                    this.onError(err);
+                } catch (e) {
+                    this.debug('Exception on this.onError function', e);
+                }
+            }
+            if (this._pendingMessages === 0) {
+                this.emit('idle');
+            }
+        });
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)